sdks/python/apache_beam/runners/portability/stager_test.py

```diff
@@ -448,7 +448,8 @@ def test_sdk_location_remote_source_file(self, *unused_mocks):
   def test_sdk_location_remote_wheel_file(self, *unused_mocks):
     staging_dir = self.make_temp_dir()
     sdk_filename = 'apache_beam-1.0.0-cp27-cp27mu-manylinux1_x86_64.whl'
-    sdk_location = 'https://storage.googleapis.com/my-gcs-bucket/' + sdk_filename
+    sdk_location = 'https://storage.googleapis.com/my-gcs-bucket/' + \
```

Comment:
Nit: implicit line joining (with parenthesis instead of backslash)  is preferable by PIP8. Since Beam does not officially use PIP8, I will merge as is, just FYI.